src/modules/airspeed_selector/AirspeedValidator.cpp

```diff
@@ -339,11 +351,21 @@ AirspeedValidator::check_first_principle(const uint64_t timestamp, const float t
 	}
 }
 
+void AirspeedValidator::checkAirspeedIncreaseDuringFrontTransition(const uint64_t timestamp,
```

Comment:
Like the other airspeed checks this one should get a bit in ASPD_DO_CHECKS to enable/disable it.